tests/test_icecream.py

```diff
@@ -518,3 +519,13 @@ def testColoring(self):
             ic({1: 'str'})  # Output should be colored with ANSI control codes.
 
         assert hasAnsiEscapeCodes(err.getvalue())
+
+    @unittest.skipIf(int(sys.version[0]) == 2, "pprint doesn't wrap long strings in Python 2.7")
+    def testLineLengthTen(self):
+        """ Test a specific line wrap width. """
+        ic.lineWrapWidth = 10
+        s = "123456789 1234567890"
+        with disableColoring(), captureStandardStreams() as (out, err):
+            ic(s)
+        pair = parseOutputIntoPairs(out, err, 2)[0]
+        self.assertEqual(pair, [('s', "('123456789 '\n '1234567890')")])
```

Comment:
I think using `parseOutputIntoPairs` makes this test worse. I suggest:

1. Checking the whole `err.getvalue()` like in `testMultilineContainerArgs`.
2. Testing the effect of `lineWrapWidth` on big structures like lists and dicts, not just strings. I expect those to still get wrapped in Python 2. `testMultilineContainerArgs` is close to doing this but it's not testing the width configuration.
3. Choosing the expected string based on the version. That should be easy to do and then the test doesn't have to be skipped.

Reply:
Done. Let me know if you want any more tests (e.g. one with a `dict`).